### ISSUE TITLE
dev mode with watcher

### ISSUE THREAD

**Issue by Alfredo-Delgado** [feathersjs/cli#163]
Is there a particular reason why a watcher and dev mode aren't included out of the box?

Would there be value in throwing this into the app generate process?

```
npm install --save-dev nodemon
```

```
scripts: {
  "dev": "nodemon src/"
}
```

**Comment by daffl** [comment]
Makes sense to me and probably fairly easily doable by adding the command to https://github.com/feathersjs/generator-feathers/blob/master/generators/app/configs/package.json.js#L31 (using `${lib}` as the folder name and `nodemon` to the `devDependencies` in https://github.com/feathersjs/generator-feathers/blob/master/generators/app/index.js#L28
